content/docs/fundamentals/converting/resolving-errors@pt_BR.md

```diff
@@ -0,0 +1,350 @@
+
+Para corrigir o erro na janela de visualização, adicione o seguinte snippet HTML à tag `<head>`:
+
+```html
+<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+```
+
+Os valores especificados para `width` e `minimum-scale` são obrigatórios na AMP. A definição de `initial-scale` não é obrigatória, mas é recomendada e costuma ser incluída no desenvolvimento da Web para dispositivos móveis. Leia mais sobre a janela de visualização e o design responsivo em [Defina a janela de visualização](https://developers.google.com/speed/docs/insights/ConfigureViewport).
+
+Assim como antes, **atualize** a página e verifique se o erro desapareceu.
+
+## Substituir folhas de estilo externas
+
+O erro a seguir está relacionado com o uso de folhas de estilo:
+
+<pre class="error-text">
+O atributo "href" na tag "link rel=stylesheet for fonts" está definido para o valor inválido "base.css".
+</pre>
+
+Especificamente, este erro se refere à seguinte tag de link da folha de estilo na tag `<head>`:
+
+```html
+<link href="base.css" rel="stylesheet" />
+```
+
+O problema é que essa é uma referência de folha de estilo externa. Na AMP, para carregar documentos com a maior velocidade possível, não inclua folhas de estilo externas. Em vez disso, todas as regras de folha de estilo precisam ser adicionadas in-line ao documento AMP usando as tags `<style amp-custom></ style>`.
+
+```html
+<style amp-custom>
+
+/* O conteúdo de base.css */
+
+</style>
+```
+
+Então, corrija o erro:
+
+1.  **Remova** a tag `<link>` que direciona para a folha de estilo na tag `<head>` e a substitua por uma tag in-line `<style amp-custom></style>`. O atributo `amp-custom` na tag de estilo é obrigatório.
+2. **Copie** todos os estilos do arquivo [`base.css`](https://github.com/googlecodelabs/accelerated-mobile-pages-foundations/blob/master/base.css) para as tags `<style amp-custom></style>`.
+
+Mais uma vez, **atualize** a página e verifique se o erro das folhas de estilo desapareceu.
+
+{% call callout('Observação', type='note') %}
+O estilo in-line é obrigatório. Além disso, há um limite de tamanho de arquivo de 50 kilobytes para todas as informações de estilo. Use pré-processadores de CSS, como o [SASS](http://sass-lang.com/) (em inglês), para reduzir o CSS antes de realizar a inserção in-line dele nas páginas AMP.
+{% endcall %}
+
+{% call callout('Importante', type='caution') %}
+É possível ter somente uma tag de estilo em todo o documento AMP. Se várias folhas de estilo externas forem referenciadas pelas páginas AMP, será necessário agrupá-las em um único conjunto de regras. Para saber quais regras de CSS são válidas na AMP, leia [CSS compatível](/pt_br/docs/design/responsive/style_pages.html).
+{% endcall %}
+
+## Excluir JavaScript de terceiros
+
+É relativamente fácil alterar as folhas de estilo inserindo CSS in-line, mas isso não se aplica ao JavaScript.
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).